internal/service/project/resource_project.go

```diff
@@ -674,9 +676,15 @@ func SetSlowOperationThresholding(ctx context.Context, performanceAdvisorAPI adm
 	return err
 }
 
-func ReadIsSlowMsThresholdingEnabled(ctx context.Context, api admin.PerformanceAdvisorApi, projectID string) (bool, error) {
+func ReadIsSlowMsThresholdingEnabled(ctx context.Context, api admin.PerformanceAdvisorApi, projectID string, warnings *diag.Diagnostics) (bool, error) {
 	response, err := api.GetManagedSlowMs(ctx, projectID).Execute()
 	if err != nil {
+		if apiError, ok := admin.AsError(err); ok && *apiError.ErrorCode == "USER_UNAUTHORIZED" {
+			if warnings != nil {
+				warnings.AddWarning("user does not have permission to read is_slow_operation_thresholding_enabled", fmt.Sprintf(ErrorProjectRead, projectID, err.Error()))
+			}
+			return false, nil
+		}
```

Comment:
example of warning:

<img width="700" alt="warning" src="https://github.com/user-attachments/assets/ebcbcfdd-452a-4e4c-9067-7bf1e6bc0ef5">